### ISSUE TITLE
The Safari extension no longer works

### ISSUE THREAD

**Issue by greggman** [benvanik/WebGL-Inspector#145]
The extension gets this error

```
[Error] XMLHttpRequest cannot load safari-extension://org.noxa.experiments.webglinspector-0000000000/ee18bc88/gli.all.js. Cross origin requests are only supported for HTTP.
	insertScript (contentscript.js:75)
	Global Code (contentscript.js:100)
[Error] NetworkError (DOM Exception 19):  A network error occurred.
	send (contentscript.js:75)
	insertScript (contentscript.js:75)
	Global Code (contentscript.js:100)
```

I have't looked into it, just thought I should report it

**Comment by benvanik** [comment]
Interesting - they must have changed the rules around requests. Chrome did the same thing awhile ago IIRC.
